src/lightmap.h

```diff
@@ -3,6 +3,7 @@
 #define LIGHTMAP_H
 
 #include <cmath>
+#include <algorithm>
```

Comment:
What's this for? You did not change anything else in this headers, so why does it suddenly need this include?
```suggestion
```

Reply:
It seemed necessary for std::min, but I’ll doublecheck :)

Reply:
`std::min` is not used in this file. If you need it in some cpp file, include it *there*. There is no need to drag this include in for *all* files that happen to include "lightmap.h".